src/signingClient.ts

```diff
@@ -56,7 +56,9 @@ import {
 } from './messages/likenft';
 import {
   formatMsgExecSendAuthorization,
+  formatMsgGrantGenericAuthorization,
   formatMsgGrantSendAuthorization,
+  formatMsgRevokeGenericAuthorization,
   formatMsgRevokeSendAuthorization,
```

Comment:
Seems no function in repo refers to `formatMsgRevokeSendAuthorization` now, could be removed?